packages/features/webhooks/components/WebhookForm.tsx

```diff
@@ -282,7 +282,7 @@ const WebhookForm = (props: {
         <Button
           type="submit"
           disabled={!formMethods.formState.isDirty && !changeSecret}
-          loading={formMethods.formState.isSubmitting || formMethods.formState.isSubmitted}>
+          loading={formMethods.formState.isSubmitting}>
```

Comment:
isSubmitted is bad here because after you submit and error happened you still will be seeing the loading.